terraform/backend.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    bucket = "forest-test-state"
+    key = "terraform.tfstate"
+    region = "us-west-1"
+    endpoint = "nyc3.digitaloceanspaces.com"
```

Comment:
Why? This should be customisable with variables. Plus, most of our infra runs on FRA1, let's stick to this region as the default. `NYC` is far for most of the team.

Reply:
Oh okay. FRA1 it is then.